LocalSettings.php

```diff
@@ -4256,8 +4258,122 @@
 	'wmgWikibaseRepoPropertyNamespaceID' => [
 		'default' => 862
 	],
+	'wmgWikibaseAllowLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseForceLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseEnableData' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseEntityAccessLimit' => [
+		'default' => null,
+		'gratispaideiawiki' => 500,
+	],
+	'wmgWikibaseAllowDataAccessInUserLanguage' => [
+		'default' => null,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseClientNamespacesWithRepoAccess' => [
+		'default' => [],
+		'gratispaideiawiki' => [
+			NS_CATEGORY,
+			NS_PROJECT,
+			NS_TEMPLATE,
+			NS_HELP,
+			828, // NS_MODULE
+		],
+	],
+	'wmgWBRepoIdGenerator' => [
+		'default' => '',
+		'gratisdatawiki' => 'mysql-upsert',
+	],
+	'wmgWBRepoIdGeneratorInErrorPingLimiter' => [
+		'default' => null,
+		'gratisdatawiki' => 9,
+	],
+	'wmgWikibaseClientEchoIcon' => [
+		'default' => [],
+		'gratispaideiawiki' => [
+			'url' => 'https://static.miraheze.org/commonswiki/a/a4/GDechoIcon.svg',
+		],
+	],
+	'wmgEchoEnablePush' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	// Only enable this ('wmgWikibaseClientTrackLuaFunctionCallsPerWiki') conservatively, having this on to many
+	// wikis will take *a lot* of Graphite storage space.
+	'wmgWikibaseClientTrackLuaFunctionCallsPerWiki' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseClientTrackLuaFunctionCallsPerSiteGroup' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseClientAddEntityUsagesBatchSize' => [
+		'default' => null,
+		'gratispaideiawiki' => 100,
+	],
+	'wmgWBRepoPreferredGeoDataProperties' => [
+		'default' => [],
+		'gratisdatawiki' => [
+			'P134',
+		],
+	],
+	'wmgWBRepoPreferredPageImagesProperties' => [
+		'default' => [],
+		'gratisdatawiki' => [
+			// Photos
+			'P386',
+			'P520',
+			'P521',
+			'P522',
+			'P523',
+			'P524',
+			// Complex graphics
+			'P135',
+			'P136',
+			'P387',
+			'P525',
+			// Simple graphics
+			'P526',
+			'P527',
+			'P528',
+			'P470',
+			'P529',
+			// Multi page content
+			'P530',
+			// Maps
+			'P531',
+			'P327',
+			'P532',
+			'P533',
+		],
+	],
 
 	// WikibaseQualityConstraints
+	'wgWBQualityConstraintsEnableConstraintsCheckJobs' => [
+		'default' => null,
+		'gratisdatawiki' => true,
+	],
+	'wgWBQualityConstraintsEnableConstraintsCheckJobsRatio' => [
+		'default' => null,
+		'gratisdatawiki' => 100,
+	],
+	'wgWBQualityConstraintsFormatCheckerShellboxRatio' => [
+		'default' => null,
+		'gratisdatawiki' => 1,
+	],
+	'wgWBQualityConstraintsTypeCheckMaxEntities' => [
+		'default' => null,
```

Comment:
Not default: https://github.com/wikimedia/mediawiki-extensions-WikibaseQualityConstraints/blob/master/extension.json#L190